packages/block-editor/src/components/rich-text/index.js

```diff
@@ -376,7 +404,7 @@ export function RichTextWrapper(
 					useFirefoxCompat(),
 					anchorRef,
 				] ) }
-				contentEditable={ true }
+				contentEditable={ ! shouldDisableEditing }
```

Comment:
A few lines down you'll need to set the tab index to:

```js
				tabIndex={
					props.tabIndex === 0 && ! shouldDisableEditing
						? null
						: props.tabIndex
				}
```

Reply:
Thanks for the tip 🙂 Done in [this commit](https://github.com/WordPress/gutenberg/pull/58085/commits/42462609211628115cd9b03cb4758228a87ab103).